scripts/index-genenetwork

```diff
@@ -549,7 +525,11 @@ def is_data_modified(xapian_directory: str,
             ])
         # Return a zero exit status code when the data has changed;
         # otherwise exit with a 1 exit status code.
-        if (db.get_metadata("generif-checksum").decode() == hash_generif_graph(sparql_uri) and
+        generif = pathlib.Path("/var/lib/data/generif-metadata.ttl")
```

Comment:
hardcoding this here makes it impossible to test locally or if our system ever changes storage paths. Can we pass this in as a cmd arg?